src/App/screens/Instructor/components/Nav/components/NavLink.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+const NavLink = ({
+  route,
+  isActive,
+  onClick,
+  href,
+}) => (
+  <a
+    href={href}
+    onClick={onClick}
+    className={`
+      f6
+      pv3 ph4 ph3-ns
+      ttu
+      no-underline
+      bl bl-0-ns bb-ns bw2 bw1-ns
+      ${isActive
+        ? 'b--orange white'
+        : 'b--transparent white-60'
+      }
```

Comment:
@trevordmiller Really a matter of preference. Typically at work we've gone with the `classNames(isActive, 'f6 pv3 ...')` style with all of the actual Tachs on one line in rough alphabetical order like you've broken out here. I like the string template way too though! 👍